server/queue/queue-activemq/src/main/java/org/apache/james/queue/activemq/ActiveMQMailQueue.java

```diff
@@ -118,14 +118,14 @@ protected void populateMailMimeMessage(Message message, Mail mail) throws Messag
         if (message instanceof BlobMessage) {
             try {
                 BlobMessage blobMessage = (BlobMessage) message;
-                try {

```

Comment:
Can someone explain me what is going one here

Reply:
not for the moment, but you can have a look. The build failed on this PR. I started to investigate, and this ugly hack is my finding. Of course it's not the solution (even if in practice it could be suitable as this attribute is never used), but it's where I am in this investigation.